test/sql/hnsw_type_checks.sql

```diff
@@ -1,6 +1,7 @@
 CREATE EXTENSION IF NOT EXISTS lanterndb;
 
 \set ON_ERROR_STOP off
+DROP TABLE IF EXISTS small_world;
```

Comment:
is this needed? doesn't `pg_regress` drop the database between tests?

Reply:
unfortunately I couldn't find any way, so `pg_regress` would drop the database after each case, one option is to iterate over test files and execute `pg_regress` for each one of them.